runner.py

```diff
@@ -143,13 +149,17 @@ def main(
     finished_pipes = start_pipe_threads(pipe_commands, logger)
 
     # Move selected files to report dir
-    logger.info(f'Moving files to {config.get("general", "report_dir")}')
-    transferred_files = report_results(finished_pipes, outdir, sample_name, config)
-    for pipe, transfers in transferred_files.items():
-        logger.info(f"Moved {transfers} files generated by {pipe}")
+    if skip_report:
+        logger.info('Reporting disabled, skipping copy to report dir')
```

Comment:
Black should replace these single quotes with double quotes (if we're going for black codestyle)